src/pages/CirclesPage/CirclesPage.tsx

```diff
@@ -37,18 +37,23 @@ export const CirclesPage = () => {
   );
   const orgs = query.data?.organizations;
 
+  const [isCircleMember, setIsCircleMember] = useState(false);
   const [showAllCircles, setShowAllCircles] = useState(false);
   const [sampleOrg, setSampleOrg] = useState<OrgWithCircles | undefined>(
     undefined
   );
-
   useEffect(() => {
     if (orgs) {
       setSampleOrg(
         orgs.find(
           o => o.sample && o.circles.length > 0 && o.created_by == profileId
         )
       );
+
+      // set isCircleMember if user is member in any circles
+      const member = orgs.some(o => o.circles.some(c => !!c.users[0]?.role));
```

Comment:
@wingfeatherwave Graffe and I cleaned this up since we didnt need the count, just a bool of isCircleMember or not.